docs/modules/postgresql.md

```diff
@@ -0,0 +1,170 @@
+# [Azure Database for PostgreSQL](https://azure.microsoft.com/en-us/services/postgresql/)
+
+|![](https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/Warning.svg/50px-Warning.svg.png) | This module is EXPERIMENTAL. It is under heavy development and remains subject to the possibility of breaking changes. |
+|---|---|
+
+Open Service Broker for Azure contains three Azure Database for PostgreSQL services. These services enable you to select the most appropriate provision scenario for your needs. These services are:
+
+| Service Name | Description ||--------------|-------------|
```

Comment:
This isn't rendering correctly. Need a line break here.

Reply:
Crazy, don't see that in code. will fix.